### PR TITLE
cryptup_reply div should be hidden in FES messages

### LINKED ISSUE #2536: cryptup_reply div should be hidden in FES messages
ref https://github.com/FlowCrypt/flowcrypt-browser/issues/5004

### PR DESCRIPTION
**Pull request by DenBond7**
This PR hides `cryptup_reply` div in FES messages

close #2536 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities

### CODE REVIEW

**Comment by sosnovsky** [comment]
@DenBond7 may be you know how I can fix this build issue:

Screenshot 2023-12-18 at 15 37 11


It occurs only in this PR, `master` branch is built without errors.

**Comment by DenBond7** [comment]
Hmm..it's a bit strange

Please be sure that you use the correct things

### Latest version
image

### consumerDebug
image

### Sync and rebuild project
image

image

**Comment by sosnovsky** [comment]
I re-checked - everything is the same as on your screenshots, but `master` builds successfully, while this PR throws error.

Maybe I need to change something in Kotlin settings:

Screenshot 2023-12-18 at 21 59 21

Error message:
```
/flowcrypt-android/FlowCrypt/build/generated/ksp/consumerDebug/java/com/flowcrypt/email/database/dao/AccountDao_Impl.java:839: error: method does not override or implement a method from a supertype
  @Override
  ^
```

On this line there is such code:

```java
 @Override
  public void addAccount(final AccountEntity accountEntity) {
    __db.beginTransaction();
    try {
      AccountDao_Impl.super.addAccount(accountEntity);
      __db.setTransactionSuccessful();
    } finally {
      __db.endTransaction();
    }
  }
```

File: FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt [hunk 1]
```diff
@@ -1008,8 +1008,15 @@ object PgpMsg {
...
-          MsgBlock.Type.PLAIN_TEXT -> msgBlock.copy(type = MsgBlock.Type.DECRYPTED_TEXT)
+          MsgBlock.Type.PLAIN_HTML -> msgBlock.copy(
+            type = MsgBlock.Type.DECRYPTED_HTML,
+            content = msgBlock.content?.let { stripFcReplyToken(it) }
+          )
+
+          MsgBlock.Type.PLAIN_TEXT -> msgBlock.copy(
+            type = MsgBlock.Type.DECRYPTED_TEXT,
+            content = msgBlock.content?.let { stripFcReplyToken(it) })
+
```
Comment (sosnovsky) [inline FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt:1019]: It's strange, but my build issue was fixed by moving closing bracket to the separate line, so
```kt
content = msgBlock.content?.let { stripFcReplyToken(it) })
```
became
```kt
  content = msgBlock.content?.let { stripFcReplyToken(it) }
)
```

Can you please update it here too

**Review by sosnovsky — COMMENTED** [review]
Works well 👍

**Comment by DenBond7** [comment]
> I re-checked - everything is the same as on your screenshots, but `master` builds successfully, while this PR throws error.
> 
> Maybe I need to change something in Kotlin settings:

I have the same settings. Maybe some plugin is having an effect. 

> AccountDao_Impl.java

It's autogenerated code. To be honest Android Studio sometimes can throw such an error that disappears by itself. It's hard to find out what exactly is wrong when you don't see Android Studio (

File: FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt (hunk 1 above)
Comment (DenBond7) [reply FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt]: Sure. Updated!

**Review by sosnovsky — APPROVED** [review]
Now builds without any issues 👍
Probably this bracket affected code generator in some way

**Pull request merged**